x-pack/plugins/code/public/components/main/content.tsx

```diff
@@ -81,9 +88,9 @@ enum ButtonOption {
 
 const Title = styled.div`
   color: #1a1a1a;
```

Comment:
If this element uses an `EuiTitle` component the color should be set automatically. Titles run a scss calculation on the `euiDarkestShade` (`#343741`) and the result is color `#1A1A1A`